automod_legacy/rules.go

```diff
@@ -238,6 +298,56 @@ OUTER:
 	return false
 }
 
+// Ticker duration for the InvitesCacheGC in minutes
+const InvitesCacheDuration = 60
+
+func inviteCacheGC() {
+	// Called right now, so we can sleep before starting
+	time.Sleep((InvitesCacheDuration * time.Minute) + (10 * time.Second))
+
+	ticker := time.NewTicker(InvitesCacheDuration * time.Minute)
+	for {
+		logger.Info("Starting invites cache GC")
+
+		start := time.Now()
+		var counter int
+
+		InvitesCache.Lock()
+
+		for guild, cache := range InvitesCache.CacheMap {
+			for i, cached := range cache {
+				if time.Since(cached.CreatedAt) > (60 * time.Minute) {
```

Comment:
Not sure if you want to use `InvitesCacheDuration` here instead of hardcoded `60`

Reply:
Yup